proto/buf/alpha/registry/v1alpha1/doc.proto

```diff
@@ -387,21 +378,28 @@ message FieldOptions {
   int32 jstype = 4;
 }
 
-// Option provides information for a descriptor option that can be show in documentations.
-message Option {
-  // The name of the option.
+// FieldLiteral provides information for a descriptor option that can be rendered in documentation.
```

Comment:
```suggestion
// FieldLiteral represents a field and its value. It can be used to model descriptor options
// and to model complex option values.
```